state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -346,6 +346,17 @@ where
 									f.ty::<TargetChainAmount<T, I>>().name("minimum_deposit")
 								}),
 						)
+					})
+					.variant(append_chain_to_name!(SetDepositChannelLifetime), |v| {
+						v.index(2).fields(
+							Fields::named()
+								.field(|f| f.ty::<TargetChainBlockNumber<T, I>>().name("lifetime")),
```

Comment:
Think i mentioned this before - it seems strange that we alter field name rather than the type name... but that's for another time..